.github/workflows/template-e2e-test/action.yaml

```diff
@@ -1,23 +1,25 @@
-# Template for e2e tests.
+# Composite action for e2e tests.
+name: Run E2E Test
+description: Run e2e test using the minikube cluster
 
 inputs:
   experiments:
     required: true
-    type: string
```

Comment:
`type` is not available.

https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs